lib/cryptoexchange/exchanges/coss/services/market.rb

```diff
@@ -17,10 +17,9 @@ def ticker_url
           "#{Cryptoexchange::Exchanges::Coss::Market::API_URL}/ticker"
         end
 
-        def adapt_all(output)
-          result = output['result']
-          result.map do |value|
-            base, target = value['MarketName'].split('-')
+        def adapt_all(result)
+          result.map do |(market_name, value)|
```

Comment:
this is not incorrect, but the codebase for block, there is no bracket for the params.

Reply:
👍